src/Microsoft.ML.CodeGen/Microsoft.ML.CodeGenerator.csproj

```diff
@@ -15,98 +18,10 @@
   <ItemGroup>
     <ProjectReference Include="..\Microsoft.ML.AutoML\Microsoft.ML.AutoML.csproj" />
   </ItemGroup>
-  
-  <ItemGroup>
```

Comment:
I don't think you want these to be removed, do you?

Reply:
bringing back